packages/components/psammead-story-promo/src/index.jsx

```diff
@@ -138,6 +158,70 @@ const InlineMediaIndicator = styled.div`
       `}
 `;
 
+/*
+ *  Text
+ */
+const TextGridColumnsTopStory = css`
+  grid-column: 1 / span 6;
+
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    grid-column: 4 / span 3;
+  }
+
+  @media (min-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN}) {
+    grid-column: 7 / span 6;
+  }
+`;
+
+const TextGridColumns = css`
+  grid-column: 3 / span 4;
+
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+    grid-column: none;
+    padding-top: ${GEL_SPACING};
+  }
+`;
+
+const TextGridFallbackTopStory = css`
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    width: ${halfWidthColumnsMaxScaleable};
+  }
+
+  @media (min-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN}) {
+    width: ${eightOfTwelveColumnsMaxScaleable};
+  }
+`;
+
+const TextGridFallback = css`
+  width: ${fourOfSixColumnsMaxWidthScaleable};
+  padding: 0 ${GEL_SPACING};
+
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+    display: block;
+    width: 100%;
+    padding: ${GEL_SPACING} 0 !important;
```

Comment:
I had to add `!important` because it was not overriding the padding when `GEL_GROUP_4_SCREEN_WIDTH_MIN` and the linting is failing on Jenkins https://ci.news.tools.bbc.co.uk/blue/organizations/jenkins/psammead/detail/story-promo-iteration/9/pipeline.
@sadick254 @dr3 Any idea how can I make the `padding: ${GEL_SPACING}` work without using important?

Reply:
Will try it locally now, you could put the padding inside a 'max width' media query so it simply stops apply?

Reply:
Also, i think it might be clashing with line 213, not 197

Reply:
Ohhh line 181's padding

Reply:
I don't think it is 181, because that one is just using `padding-top`. It applies the following
![Screenshot 2019-10-03 at 11 42 45](https://user-images.githubusercontent.com/46446236/66120516-ef2b5c80-e5d2-11e9-8224-6d5f0080b5ac.png)

Reply:
Discussed on slack. The issue was the order of the media queries.